ci/slack_user_ids

```diff
@@ -5,3 +5,4 @@ shayne.fletcher@digitalasset.com UBRBXAJHY
 shayne@shaynefletcher.org UBRBXAJHY
 stefano.baghino@digitalasset.com UD3MUK6TA
 nick.chapman@digitalasset.com UHHMLCNGM
+remy.haemmerl@digitalasset.com UEGT3JT0Q
```

Comment:
```suggestion
remy.haemmerle@daml.com UEGT3JT0Q
```
This needs to be the email address you are committing with. Looking at `git log` it seems to be this one.